src/screens/NationDetailsContainer/index.js

```diff
@@ -99,6 +99,16 @@ class NationDetailsContainer extends NavigatorComponent {
     this.performIfHasWallet(this.props.leaveNation);
   };
 
+  openNationChat = () => {
+    const id = openedNation(this.props).id;
```

Comment:
this.props.openedNationId can be used instead, since it's the purpose of that field and actually openedNation() takes nation based on openedNationId.